cmd/rokka/cli/cmd_massupload.go

```diff
@@ -0,0 +1,161 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+	"fmt"
+	"errors"
+	"os"
+	"sync"
+	"github.com/rokka-io/rokka-go/cmd/rokka/cli/massupload"
+	"github.com/rokka-io/rokka-go/rokka"
+	"path/filepath"
+	// TODO: "syscall"
+	// TOOD: "os/signal"
+)
+
+type MassuploadOptions struct {
+	Organization string
+	BasePath string
+	Recursive bool
+	DryRun bool
+	Extensions []string
+	Concurrency int
+}
+
+var massuploadCmd = &cobra.Command{
+	Use:                   "massupload [organization] [path]",
+	Short:                 "Upload all images from a folder to rokka",
+	Args:                  cobra.ExactArgs(2),
+	// Aliases:               []string{"c"},
+	DisableFlagsInUseLine: true,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		massuploadOptions.Organization = args[0]
+		massuploadOptions.BasePath = args[1]
+
+		return massuploadOptions.validate()
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runMassUpload(rokkaClient, massuploadOptions)
+	},
```

Comment:
please use the same way we do it everywhere: using the `run` function and a template

Reply:
Hum, how would you handle the template with a "running" output?

Reply:
hm, right. But you could still do it using a template, to make each output "configurable", no?

Reply:
Added configurable templates for:
- command output
- successful upload item
- error item